js_modules/dagster-ui/packages/ui-core/src/instance/InstanceBackfills.tsx

```diff
@@ -124,9 +85,9 @@ export const InstanceBackfills = () => {
               <span>Backfills are moving:</span>
               <span style={{fontWeight: 'normal'}}>
                 {' '}
-                We&apos;re incorporating backfills into the <Link to="/runs/">Runs</Link> page to
-                unify the UI and provide one page to see all of your executions. The Backfills page
-                will be removed in a future release.{' '}
+                We&apos;re moving backfills to the <Link to="/runs?view=BACKFILLS">Runs</Link> page
```

Comment:
Can this be a lowercase query arg value, maybe converted to uppercase under the hood for GraphQL? It's awfully loud as uppercase.

Reply:
Hmm that's a good idea, I can add a transform to the useQueryPersistedState for this